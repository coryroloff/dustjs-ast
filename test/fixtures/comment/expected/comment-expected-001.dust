{! {?exists}{@helper param="hello"/}, {world}!{/exists} !}