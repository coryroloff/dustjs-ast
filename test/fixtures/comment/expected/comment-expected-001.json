{
	"type": "Template",
	"body": [
		{
			"type": "Comment",
			"value": " {?exists}{@helper param=\"hello\"/}, {world}!{/exists} ",
			"start": 0,
			"end": 58,
			"loc": {
				"start": {
					"line": 1,
					"column": 0
				},
				"end": {
					"line": 1,
					"column": 58
				}
			}
		}
	],
	"tokens": [
		{
			"type": "Comment",
			"value": " {?exists}{@helper param=\"hello\"/}, {world}!{/exists} ",
			"start": 0,
			"end": 58,
			"loc": {
				"start": {
					"line": 1,
					"column": 0
				},
				"end": {
					"line": 1,
					"column": 58
				}
			}
		}
	],
	"start": 0,
	"end": 58,
	"loc": {
		"start": {
			"line": 1,
			"column": 0
		},
		"end": {
			"line": 1,
			"column": 58
		}
	}
}